src/panoramic/cli/cli.py

```diff
@@ -211,3 +211,112 @@ def scaffold(target_dataset: str, yes: bool):
     )
 
     scaffold_missing_fields_command(target_dataset=target_dataset, yes=yes)
+
+
+@cli.group()
+def data_connections():
```

Comment:
Very cool, especially the thorough doc block. I'd maybe consider changing the group name to `connections`.

Reply:
done